### PR TITLE
BITMAKER-2057 New Login & Register frontend implemented

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

estela Frontend new design implementation Login

# Issue

https://tasks.bitmaker.dev/issues/2057?tab=time_entries

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@Adriana618 Could you check this PR. I think is done!

**Comment by raymond1242** [comment]
@Adriana618 Now it's ready

**Comment by raymond1242** [comment]
@Adriana618 Could you review this last commit pls

**Review by Adriana618 — CHANGES_REQUESTED** [review]
Looks good, just I think we need to center the view. Both, in login and register.

**Comment by raymond1242** [comment]
@Adriana618 Now its centered. Done!

File: estela-web/src/components/LoginPage/index.tsx [hunk 1]
```diff
@@ -38,34 +37,60 @@ export class LoginPage extends Component<unknown> {
...
+                                    rules={[{ required: true, message: "Please input your password" }]}
+                                >
+                                    <Input.Password autoComplete="current-password" className="border-[#4D47C3]" />
+                                </Form.Item>
+                            </div>
+                            <Button
+                                block
+                                htmlType="submit"
+                                className="border-[#4D47C3] bg-[#4D47C3] hover:border-[#4D47C3] hover:text-[#4D47C3] text-white rounded-md text-sm"
+                            >
```
Comment (Adriana618) [inline estela-web/src/components/LoginPage/index.tsx:80]: Don't hardcode colors, instead create a class in a .scss file

**Review by Adriana618 — CHANGES_REQUESTED** [review]

**Review by Adriana618 — APPROVED** [review]

**Pull request merged**